pkg/cluster/context.go

```diff
@@ -178,6 +178,18 @@ func (c *Context) Delete() error {
 	if err != nil {
 		return fmt.Errorf("error listing nodes: %v", err)
 	}
+
+	// try to remove the kind kube config file generated by "kind create cluster"
+	err = os.Remove(c.KubeConfigPath())
+	if err != nil {
+		fmt.Printf("Tried to remove %s but recieved error: %s\n", c.KubeConfigPath(), err)
+	}
+
+	// check if $KUBECONFIG is set and let the user know to unset if so
+	if os.Getenv("KUBECONFIG") == c.KubeConfigPath() {
+		fmt.Printf("$KUBECONFIG is still set to use %s even though that file has been deleted, remember to unset it to get back to your original contexts\n", c.KubeConfigPath())
```

Comment:
Also, may be get rid of a the part mentioning contexts. 
To my understanding, a context is a combination of access parameters - it
Takes into account user accounts and namespaces as well, not just clusters.

Reply:
yeah, contexts is orthogonal to entire config files, also if your original contexts were in the default file you don't even need to do this, and if they weren't, unsetting it won't be sufficient, you'll need to re-set it to wherever those were

Reply:
Are you in favor of erring on the side of verbosity and saying "remember to unset it to use the default kube config, or set it back to your original value", or just saying "remember to unset it"?

Reply:
I think just "remember to unset it", we should probably discuss in a follow-up how to improve the message from there. 
EG handling windows is probably something we _should_ do but more complex than the first PR needs to be 😅 

We should look around at other tools for guidance 🙃